Data.Service/ProjectService.cs

```diff
@@ -0,0 +1,149 @@
+﻿using Data.Entities;
+using Data.EntitiyFrameWork;
+using Data.ViewModels;
+using Data.ViewModels.ViewModels;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+
+namespace Data.Service
+{
+    public class ProjectService : IProjectService
+    {
+        private readonly DataBase1Context dbContext;
+        public ProjectService(DataBase1Context dbContext)
+        {
+            this.dbContext = dbContext;
+        }
+        public int Add(ProjectViewModel Entity, string UserName)
+        {
+            var project = new Project();
+            project.Title = Entity.Title;
+            project.Created = Entity.Created;
+            project.Subject = Entity.Subject;
+            project.ExpirationDate = Entity.ExpirationDate;
+
```

Comment:
Use object initializer (https://www.c-sharpcorner.com/UploadFile/mahesh/object-initializer-in-C-Sharp/)